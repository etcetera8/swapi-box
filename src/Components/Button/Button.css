.Button {
  margin: 10px;
  padding: 3px;
  width: 125px;
  border: 2px solid #F1D441;
  background: none;
  color: #f3da4a;
  font-size: 1.2em;
  letter-spacing: 1px;
  transition: box-shadow 0.2s ease-in-out;
}

.Button:focus {
  outline: 0;
}

.Button:hover {
  box-shadow: 0 0 8px;
}

.activevehicles, 
.activepeople,
.activeplanets,
.activefavorites {
  box-shadow: 0 0 8px;
  border: 2px solid #AD1F1E;
  color: #AD1F1E;
}
