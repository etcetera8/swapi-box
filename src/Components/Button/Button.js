import React from 'react';
import './Button.css';

const Button = () => {
  return (
    <button className='Button'> 
      button
    </button>
  )
}

export default CardContainer;