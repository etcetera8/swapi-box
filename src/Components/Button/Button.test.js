import React from 'react';
import ReactDOM from 'react-dom';
import Button from './Button';
import { shallow } from 'enzyme';

describe('Button', () => {
  let wrapper;
  const mockFunction = jest.fn();
  const mockArray = [{name: 'luke', homeworld:'naboo', species:'human', population: '20'}]
  
  beforeEach( () => {
    wrapper = shallow(<Button
                        name={'people'}
                        resourceCall={mockFunction}
                        className={'peopleActive'}
                      />)
  })

  it('should match the snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  })
})