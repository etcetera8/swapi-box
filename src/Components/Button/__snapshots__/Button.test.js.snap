// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot test 1`] = `
<button
  className="Button peopleActive"
  onClick={[Function]}
>
  people
</button>
`;
